js/src/components/SchemaTab.js

```diff
@@ -0,0 +1,234 @@
+	{
+		name: __( "Web Page", "wordpress-seo" ),
+		value: "option-1",
+	},
+	{
+		name: __( "Item Page", "wordpress-seo" ),
+		value: "option-2",
+	},
+	{
+		name: __( "About Page", "wordpress-seo" ),
+		value: "option-3",
+	},
+	{
+		name: __( "FAQ Page", "wordpress-seo" ),
+		value: "option-4",
+	},
+	{
+		name: __( "QA Page", "wordpress-seo" ),
+		value: "option-5",
+	},
+	{
+		name: __( "Profile Page", "wordpress-seo" ),
+		value: "option-6",
+	},
+	{
+		name: __( "Contact Page", "wordpress-seo" ),
+		value: "option-7",
+	},
+	{
+		name: __( "Medical Web Page", "wordpress-seo" ),
+		value: "option-8",
+	},
+	{
+		name: __( "Collection Page", "wordpress-seo" ),
+		value: "option-9",
+	},
+	{
+		name: __( "Checkout Page", "wordpress-seo" ),
+		value: "option-10",
+	},
+	{
+		name: __( "Real Estate Listing", "wordpress-seo" ),
+		value: "option-11",
+	},
+	{
+		name: __( "Search Results Page", "wordpress-seo" ),
+		value: "option-12",
+	},
+	{
+		/* translators: %1$s expands to "- " (a hyphen and a space), %2$s expands to " -" (a space and a hyphen) */
+		name: sprintf( __( "%1$sNone%2$s", "wordpress-seo" ), "- ", " -" ),
+		value: "option-13",
+	},
+];
+
+const schemaArticleOptions = [
+	{
+		name: __( "Article", "wordpress-seo" ),
+		value: "option-1",
+	},
+	{
+		name: __( "Social Media Posting", "wordpress-seo" ),
+		value: "option-2",
+	},
+	{
+		name: __( "News Article", "wordpress-seo" ),
+		value: "option-3",
+	},
+	{
+		name: __( "Advertiser Content Article", "wordpress-seo" ),
+		value: "option-4",
+	},
+	{
+		name: __( "Satirical Article", "wordpress-seo" ),
+		value: "option-5",
+	},
+	{
+		name: __( "Scholary Article", "wordpress-seo" ),
+		value: "option-6",
+	},
+	{
+		name: __( "Tech Article", "wordpress-seo" ),
+		value: "option-7",
+	},
+	{
+		name: __( "Report", "wordpress-seo" ),
+		value: "option-8",
+	},
+	{
+		/* translators: %1$s expands to "- " (a hyphen and a space), %2$s expands to " -" (a space and a hyphen) */
+		name: sprintf( __( "%1$sNone%2$s", "wordpress-seo" ), "- ", " -" ),
+		value: "option-9",
+	},
+];
+
+/* translators: %1$s and %2$s expand to a link to the Search Appearance Settings page */
+const footerText = sprintf(
+	__( "You can change the default type for Posts in your %1$sSearch Appearance Settings%2$s.", "wordpress-seo" ),
+	"{{link}}",
+	"{{/link}}"
+);
+
+const footerWithLink = interpolateComponents(
+	{
+		mixedString: footerText,
+		// eslint-disable-next-line jsx-a11y/anchor-has-content
+		components: { link: <a href="/wp-admin/admin.php?page=wpseo_titles#top#post-types" /> },
+	},
+);
+
+/**
+ * Returns the content of the schema tab.
+ *
+ * @param {object} props Component props.
+ *
+ * @returns {React.Component} The schema tab content.
+ */
+const Content = ( props ) => (
+	<Fragment>
+		<div className="yoast-field-group__title">
+			<b>{ props.helpTextTitle }</b>
+			<HelpIcon
+				linkTo={ props.helpTextLink }
+				linkText={ __( "Ultimate structured data guide", "wordpress-seo" ) }
```

Comment:
This will be set as the `screen-reader-text`. The screen-reader-text below on line 149 is very descriptive, whereas this one is just the title of the Article being linked to.

Reply:
I have changed this text to "Learn more about structured data with Schema.org" as this provides a little more context to screenreaders.